src/pages/home/Home.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+import "./home.css";
+import ArticleCard from "../../components/ArticleCard/articleCard/ArticleCard";
+
+const Home = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetch("https://hstestncnews.herokuapp.com/api/articles")
+      .then((response) => response.json())
+      .then(({ articles }) => {
+        setArticles(articles);
+      });
+  }, []);
```

Comment:
Could you refactor this so all your api requests are in the same file i.e. api.js?
like the lecture on monday